src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
 import './ChatEntry.css';
+
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const changeHeart = () => {
+    props.toggleHeartCallback(props.id);
+  };
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div
+      className={`chat-entry ${
+        props.sender === 'Estragon' ? 'remote' : 'local'
+      }`}
+    >
```

Comment:
Good use of the ternary operator! Also good way to distinguish the remote user from the local user!

Further, I think within the hierarchy this is exactly correct place for this check. It wouldn't make sense higher up.